tests/unit/metadata_manager_test.py

```diff
@@ -177,82 +180,64 @@ def test_extract_meta_data(metadata_manager: MetadataManager):
 """
 
 
-def test_start(mocker, metadata_manager: MetadataManager):
+def test_start(metadata_manager: MetadataManager):
     cache_manager = get_cache_manager()
 
-    test_host = "test_host"
     cache_manager._hosts = []
-    cache_manager.domain = test_host
+    cache_manager.domain = "google.com"
+
+    message = Message(
+        extractors=None,
+        bypass_cache=False,
+        html=None,
+        header=None,
+        url="https://google.com",
+        _shared_memory_name="test",
+        har={},
+    )
 
-    message = {
-        MESSAGE_SHARED_MEMORY_NAME: "test",
-        MESSAGE_URL: "empty_url",
-        MESSAGE_BYPASS_CACHE: "f",
-        MESSAGE_ALLOW_LIST: {},
-        MESSAGE_HTML: "test_html",
-        MESSAGE_HEADERS: "",
-    }
-    log_spy = mocker.spy(metadata_manager, "_logger")
+    with open(
+        Path(__file__).parent.parent / "splash-response-google.json", "r"
+    ) as f:
+        splash_response = json.load(f)
+
+    async def accessibility_api_call_mock(
+        self, website_data, session, strategy
+    ) -> float:  # noqa
+        return 0.98
```

Comment:
For the accessibility, you could also check whether this value is actually returned.